src/config/settings.ts

```diff
@@ -118,6 +118,102 @@ export async function clear() {
   await fs.promises.rm(paths.config, { recursive: true, force: true } as any);
 }
 
+function getConfigLHS(cfg: any, fqFieldName: string): [any, string] | null {
```

Comment:
This function is doing complicated enough things that it needs documentatino. _What_ `LHS` do are you getting? What the heck is `cfg` (given that there's no typing to help understanding)? What are you returning?

Reply:
I added more docs and shrunk the code (with better typing).